<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>5/2/0</td>
</tr><tr>
<td>Started on:</td><td>Mon Jul 28 20:44:20 IST 2025</td>
</tr>
<tr><td>Total time:</td><td>80 seconds (80506 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testCases.TC003_Login_DD.logintest()'><b>logintest</b><br>Test class: testCases.TC003_Login_DD<br>Parameters: rp@gmail.com, , Valid</td>
<td><div><pre>java.lang.AssertionError: Login failed, but was expected to succeed.
	at testCases.TC003_Login_DD.logintest(TC003_Login_DD.java:52)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace925913781", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace925913781'><pre>java.lang.AssertionError: Login failed, but was expected to succeed.
	at org.testng.Assert.fail(Assert.java:111)
	at testCases.TC003_Login_DD.logintest(TC003_Login_DD.java:52)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>11</td>
<td>testCases.TC003_Login_DD@576f63f6</td></tr>
<tr>
<td title='testCases.TC003_Login_DD.logintest()'><b>logintest</b><br>Test class: testCases.TC003_Login_DD<br>Parameters: rpn@gmail.com, test@123, Invalid</td>
<td><div><pre>java.lang.AssertionError: Login succeeded, but was expected to fail.
	at testCases.TC003_Login_DD.logintest(TC003_Login_DD.java:62)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1515282013", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1515282013'><pre>java.lang.AssertionError: Login succeeded, but was expected to fail.
	at org.testng.Assert.fail(Assert.java:111)
	at testCases.TC003_Login_DD.logintest(TC003_Login_DD.java:62)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at org.testng.TestRunner.privateRun(TestRunner.java:741)
	at org.testng.TestRunner.run(TestRunner.java:616)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
	at org.testng.SuiteRunner.run(SuiteRunner.java:312)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
	at org.testng.TestNG.runSuites(TestNG.java:1112)
	at org.testng.TestNG.run(TestNG.java:1079)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
</pre></div></td>
<td>15</td>
<td>testCases.TC003_Login_DD@576f63f6</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td><b>Attribute(s)</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='testCases.TC003_Login_DD.logintest()'><b>logintest</b><br>Test class: testCases.TC003_Login_DD<br>Parameters: rp@gmail.com, rp@1234987, Invalid</td>
<td></td>
<td>11</td>
<td>testCases.TC003_Login_DD@576f63f6</td></tr>
<tr>
<td title='testCases.TC001_AccountRegistration.verify_account_registration()'><b>verify_account_registration</b><br>Test class: testCases.TC001_AccountRegistration</td>
<td></td>
<td>2</td>
<td>testCases.TC001_AccountRegistration@2a640157</td></tr>
<tr>
<td title='testCases.TC002_Login.logintest()'><b>logintest</b><br>Test class: testCases.TC002_Login</td>
<td></td>
<td>2</td>
<td>testCases.TC002_Login@5d8bafa9</td></tr>
<tr>
<td title='testCases.TC003_Login_DD.logintest()'><b>logintest</b><br>Test class: testCases.TC003_Login_DD<br>Parameters: , rp@1234, Invalid</td>
<td></td>
<td>12</td>
<td>testCases.TC003_Login_DD@576f63f6</td></tr>
<tr>
<td title='testCases.TC003_Login_DD.logintest()'><b>logintest</b><br>Test class: testCases.TC003_Login_DD<br>Parameters: rpn@gmail.com, test@123, Valid</td>
<td></td>
<td>12</td>
<td>testCases.TC003_Login_DD@576f63f6</td></tr>
</table><p>
</body>
</html>